relay-event-schema/src/protocol/span.rs

```diff
@@ -76,6 +76,12 @@ pub struct Span {
     /// Timestamp when the span has been received by Sentry.
     pub received: Annotated<Timestamp>,
 
+    /// Temporary protocol support for metric summaries.
+    ///
+    /// This shall move to a stable location once we have stabilized the
+    /// interface.  This is intentionally not typed today.
+    pub _metrics_summary: Annotated<Object<Value>>,
+
```

Comment:
Properties on spans are retained anyway (see `additional_properties` with `retain=true` below), so this addition is probably not even necessary.

Reply:
Good catch, I missed that when talking to Armin!